src/dbsync/src/dbsync_implementation.h

```diff
@@ -45,23 +45,79 @@ namespace DbSync
                                  const DbEngineType dbType,
                                  const std::string& path,
                                  const std::string& sqlStatement);
+        TXN_HANDLE createTransaction(const DBSYNC_HANDLE handle,
+                                     const char** tables,
+                                     const int threadNumber,
+                                     const int maxQueueSize,
+                                     result_callback_t callback);
+        void closeTransaction(const DBSYNC_HANDLE handle,
+                              TXN_HANDLE txnHandle);
         void release();
     private:
-        struct DbEngineContext
+
+        struct TransactionContext
+        {
+            explicit TransactionContext(const char** tables)
+            {
+                auto rawElement { *tables };
+                if (nullptr != rawElement)
+                {
+                    while ('\0' != *rawElement)
+                    {
+                        m_tables.push_back(std::string(rawElement));
+                        rawElement += m_tables.back().size() + 1;
+                    }
+                } 
+                else 
+                {
+                    throw dbsync_error
+                    {
+                        INVALID_TRANSACTION
+                    };
+                }
+            }
+            std::vector<std::string> m_tables;
```

Comment:
I think this should be called tables, although is not a POD it would be better to use that naming convention.